### PR TITLE
Update name of Robotect EV3 extension

### PR DESCRIPTION
**Pull request by khanning**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -184,6 +184,23 @@ <h2 class="logo-scratchx">ScratchX</h2>
                         </section>
                     </li>
                     <li>
+                        <a href="?url=https://Robotec-CoderZ.github.io/he.js" data-action="load-url">
+                            <header>
+                                <h2>EV3 LEGO Education</h2>
```
Comment (thisandagain) [inline index.html]: I find it confusing that we now have two extensions that at a surface level appear to do the same thing. I recommend that we do the following:

- Rename the existing "EV3+Scratch" extension to "LEGO Mindstorms EV3 (Bluetooth)"
- Rename this extension to "LEGO Mindstorms EV3 (USB)"

**Review by thisandagain — DISMISSED** [review]

File: index.html (hunk 1 above)
Comment (khanning) [reply index.html]: This sounds good to me. I'll send a courtesy email to the developers to confirm the name changes.

Thanks!

Comment (khanning) [reply index.html]: The only other source of confusion I see is that the USB extension is currently only in Hebrew. This is currently indicated in the extension description.

Comment (thisandagain) [reply index.html]: True. I wonder if we could ask the author to signal that more clearly in the graphic?

**Comment by thisandagain** [comment]
@khanning Any updates on this?

**Comment by khanning** [comment]
@thisandagain the developers of the USB (Hebrew) version did not like the proposed name change. To them, highlighting the fact that the extension supports Hebrew is more important than the connection protocol. They offered two alternatives:
- EV3+הגרסה העברית: סקראצ' *(If we are willing to have the name of the extension be in Hebrew)*
-or-
- EV3+Scratch Hebrew Version

**Review by khanning — COMMENTED** [review]
Changed name to "EV3+Scratch (Hebrew Version)"

**Pull request merged**